### ISSUE TITLE
Cannot Start VirtualBox on Ubuntu 14.04

### ISSUE THREAD

**Issue by anonRegions** [ideasonpurpose/basic-wordpress-vagrant#21]
Host OS: Ubuntu 14.04
Guest OS: 'Ubuntu_64'
VirtualBox: 5.0.16 r105871
Vagrant: 1.7.4

Attempting to start up VirtualBox & vagrant this morning via `vagrant up`, I got the following error message:

Command: ["hostonlyif", "create"]

Stderr: 0%...
Progress state: NS_ERROR_FAILURE
VBoxManage: error: Failed to create the host-only adapter
VBoxManage: error: VBoxNetAdpCtl: Error while adding new interface: failed to open /dev/vboxnetctl: No such file or directory
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component HostNetworkInterfaceWrap, interface IHostNetworkInterface
VBoxManage: error: Context: "RTEXITCODE handleCreate(HandlerArg*)" at line 71 of file VBoxManageHostonly.cpp

I then did the following:

`sudo /Library/StartupItems/VirtualBox/VirtualBox restart`

yielding this error message:

"sudo: /Library/StartupItems/VirtualBox/VirtualBox: command not found"

Attached is the relevant vbox.log:

[140704_vboxlog.txt](https://github.com/ideasonpurpose/basic-wordpress-vagrant/files/211412/140704_vboxlog.txt)

Not sure if you need my vagrantfile, but here are the contents of that:

[Vagrantfile.txt](https://github.com/ideasonpurpose/basic-wordpress-vagrant/files/211419/Vagrantfile.txt)

Trying to restart VirtualBox failed:

`sudo /etc/init.d/vboxdrv restart
`
yields this message:

Stopping VirtualBox kernel modules ...done.
Starting VirtualBox kernel modules ...failed!
(modprobe vboxnetflt failed. Please use 'dmesg' to find out why)

I was then informed that the issue is entirely a VirtualBox issue. 

I ran this command:

`$ dmesg | grep VirtualBox
`
getting this result:

[ 5067.037574] warning: `VirtualBox' uses 32-bit capabilities (legacy
support in use)

Then I ran

`sudo modprobe -v vboxdrv
`
which did absolutely nothing after I entered the password at the prompt:

`$ sudo modprobe -v vboxdrv
`
[sudo] password for selfspunwebs:

Trying this:

`$ sudo apt-get install linux-headers-generic build-essential
`
got me this:

Reading package lists... Done
Building dependency tree
Reading state information... Done
build-essential is already the newest version.
linux-headers-generic is already the newest version.
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.

Then I tried this:

`sudo depmod -a
`
and then modprobe again:

`sudo modprobe -v vboxdrv
`
which again yielded nothing.

The moderator of the virtualbox.org forum on VirtualBox for Linux suggests that I install a Linux guest like mint in VirtualBox for testing purposes. Before I do that, I was wondering if you have any feedback for me on the above.

Please advise.

Thanks.

**Comment by joemaller** [comment]
I don't think it's a problem specific to our Vagrantfile, but were you able to get it working again? Can you spin up a new vagrant instance from a fresh clone/download in a different directory?

**Comment by anonRegions** [comment]
I haven't been able to get it to work & I've tried from two or three
directories.

I'm mystified as to why apt-get keeps grabbing vagrant 1.7.4, when I want
1.8.1. What's that about?

On Wednesday, April 13, 2016, Joe Maller notifications@github.com wrote:

> I don't think it's a problem specific to our Vagrantfile, but were you
> able to get it working again? Can you spin up a new vagrant instance in a
> different directory?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/ideasonpurpose/basic-wordpress-vagrant/issues/21#issuecomment-209444775

## 

beyondtheframe@gmail.com

Ethan Rips
beyondtheframe@gmail.com
https://www.linkedin.com/in/ethanrips

**Comment by joemaller** [comment]
Have you tried Vagrantfiles from other projects? This sounds like a bigger Vagrant or VirtualBox issue. I haven't ever used Vagrant on a Linux desktop, so I'm afraid I can't be much help there.

**Comment by anonRegions** [comment]
Joe--

I have not.

Would you by any chance have specific instructions for me on how to install
vagrant? I'm thinking I must be missing something.

Btw, I neglected to give you full information on my setup. Here goes:

Host OS: Ubuntu 14.04
Guest OS: Linux Mint 17.1
VirtualBox: 5.0.16 r105871
Vagrant: 1.7.4

The guest works--I can see Mint running in VirtualBox.

The problem is that I need vagrant as a precursor to installing vvv & when
I try to install vagrant, I get version 1.7.4, then an error message
telling me 1.7.4 is incompatible with VirtualBox 5.0. What's really absurd
about this is that I was using VirtualBox 5.0 before the VM got corrupted.

Please advise.

vbox.log & vbox-install.log are attached.

Thanks.

Ethan

beyondtheframe@gmail.com

Ethan Rips
beyondtheframe@gmail.com
https://www.linkedin.com/in/ethanrips

**Comment by anonRegions** [comment]
Joe--

Cancel previous transmissions. I figured it out & installed vagrant 1.8.1.

Next question: My next task is to initialize vagrant in a directory. Is
there a specific directory where I should do this? Previously I navigated
to vagrant-local to start up vagrant, which makes sense, given that that is
where I'll house my WordPress sites. I have no idea whether vagrant or
anything else will recognize or play well with my WordPress sites since the
reinstalls. Any feedback on that?

Thanks.

Ethan

beyondtheframe@gmail.com

Ethan Rips
beyondtheframe@gmail.com
https://www.linkedin.com/in/ethanrips
